concurrent.lib.sh

```diff
@@ -315,6 +315,7 @@ concurrent() (
             while read -r meta; do
                 printf "meta:%d:%s\n" "${1}" "${meta}" >> "${4}"
             done
+            exit $PIPESTATUS
```

Comment:
Nice catch! Shellcheck is complaining here, though:

```
In concurrent.lib.sh line 318:
            exit $PIPESTATUS
                 ^-- SC2128: Expanding an array without an index only gives the first element.
                 ^-- SC2086: Double quote to prevent globbing and word splitting.
```

I think this is what we want:

```bash
exit "${PIPESTATUS[0]}"
```